### PR TITLE
Update CD pipelines with new modules

### PR DESCRIPTION
**Pull request by r00ta**
[Insert JIRA Issue number and update link](https://issues.redhat.com/browse/MGDOBR-) 

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: .github/workflows/CD.yml [hunk 1]
```diff
@@ -71,11 +71,11 @@ jobs:
         with:
           filters: |
             general: &general
+              - actions/**
               - infra/**
               - bom/**
               - build-parent/**
-              - infra/**
```
Comment (sutaakar) [inline .github/workflows/CD.yml:77]: Why is the `infra` removed?

Comment (r00ta) [reply .github/workflows/CD.yml:77]: Because it was duplicated, see line 74

Comment (r00ta) [reply .github/workflows/CD.yml:77]: A typo I guess

Comment (sutaakar) [reply .github/workflows/CD.yml:77]: I see

**Review by sutaakar — APPROVED** [review]

**Pull request merged**